tdvt/tdvt/exprtests/standard/expected.setup.operator.date.10.txt

```diff
@@ -94,13 +94,13 @@
         <value>-10491.333</value>
       </tuple>
       <tuple>
-        <value>-85.428888</value>
+        <value>-85.428889</value>
```

Comment:
These values have changed because of that rounding issue you talked about in standup, right?

Reply:
Is this file only used by Tibco? It might be safer to add another expected if you aren't sure.